sdk/communication/azure-communication-rooms/src/samples/java/com/azure/communication/rooms/ReadmeSamples.java

```diff
@@ -61,6 +61,13 @@ public RoomsClient createRoomsClientWithConnectionString() {
         return roomsClient;
     }
 
+    public RoomsClientBuilder createRoomsClientBuilder() {
+        // BEGIN: readme-sample-createRoomsCLientBuilder
+        RoomsClientBuilder builder = new RoomsClientBuilder();
```

Comment:
Is this used in the readme?

Reply:
No but used in the client builder java doc comment.